model/internal/data/optional_double.go

```diff
@@ -0,0 +1,80 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package data // import "go.opentelemetry.io/collector/model/internal/data"
+
+import (
+	"github.com/gogo/protobuf/types"
```

Comment:
Hey @bogdandrutu ,

This project is currently unmaintained, is there any reason why not to use https://github.com/golang/protobuf ?

Reply:
@MovieStoreGuy you can see more details on the discussion about what to do w/ gogo in the proto repo PR https://github.com/open-telemetry/opentelemetry-proto/pull/336

Basically it comes down to performance, which is why we're investigating alternatives.